tools/tests_sorted/tests_sorted.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"fmt"
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// file paths to ignore
+var ignorePaths = []string{ //nolint:gochecknoglobals
+	// Need fixing.
+	"internal/cli/dialog/components/checklist_test.go",
+	"internal/cli/dialog/components/test_input_test.go",
+	"internal/cli/format/indent_test.go",
+	"internal/cmd/completions_test.go",
+	"internal/config/configdomain/dryrun_test.go",
+	"internal/config/configdomain/hosting_platform_test.go",
+	"internal/config/configdomain/lineage_test.go",
+	"internal/config/configdomain/ship_delete_tracking_branch_test.go",
+	"internal/config/configdomain/sync_strategy_test.go",
+	"internal/git/commands_test.go",
+	"internal/git/gitdomain/branch_infos_test.go",
+	"internal/git/gitdomain/branch_name_test.go",
+	"internal/git/gitdomain/local_branch_names_test.go",
+	"internal/git/gitdomain/shas_test.go",
+	"internal/gohacks/error_collector_test.go",
+	"internal/gohacks/slice/append_all_missing_test.go",
+	"internal/gohacks/slice/contains_any_test.go",
+	"internal/gohacks/slice/find_many_test.go",
+	"internal/gohacks/slice/get_all_test.go",
+	"internal/gohacks/slice/hoist_test.go",
+	"internal/gohacks/slice/remove_at_test.go",
+	"internal/gohacks/slice/remove_test.go",
+	"internal/gohacks/slice/truncate_last_test.go",
+	"internal/gohacks/stringslice/collector_test.go",
+	"internal/gohacks/stringslice/lines_with_prefix_test.go",
+	"internal/hosting/detect_test.go",
+	"internal/undo/undobranches/branch_changes_test.go",
+	"internal/undo/undobranches/branch_span_test.go",
+	"internal/undo/undoconfig/config_test.go",
+	"internal/vm/optimizer/remove_duplicate_checkout_test.go",
+	"internal/vm/program/program_test.go",
+	"pkg/prelude/option_test.go",
+	"test/datatable/branch_setup_table_test.go",
+	"test/datatable/data_table_test.go",
+	"test/filesystem/copy_directory_test.go",
+	"test/fixture/factory_test.go",
+	"test/fixture/fixture_test.go",
+	"test/output/git_town_output_parser_test.go",
+	"test/subshell/test_runner_test.go",
+	"test/testgit/locations_test.go",
+	"test/testruntime/testruntime_test.go",
+	"tools/format_unittests/format_unittests_test.go",
+	// Not owned.
+	"vendor/",
+}
+
+func main() {
+	issues := Issues{}
+	err := filepath.Walk(".", func(path string, info os.FileInfo, err error) error {
+		if err != nil || info.IsDir() || !strings.HasSuffix(info.Name(), ".go") || isIgnoredPath(path) {
+			return err
+		}
+		fileIssues, err := LintFile(path, nil)
+		if err != nil {
+			return err
+		}
+		issues = append(issues, fileIssues...)
+		return nil
+	})
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+	if len(issues) > 0 {
+		fmt.Fprintln(os.Stderr, issues)
```

Comment:
Nitpick: Given that the point of a linter is to print lint warnings, one could argue that these lint warnings are the regular output of this tool and should therefore go to STDOUT. STDERR is for when the linter itself doesn't work properly. What do you think?

Reply:
Done.